src/pe/load_config.rs

```diff
@@ -336,8 +336,8 @@ impl LoadConfigData {
                         dd.virtual_address
                     ))
                 })?;
-        let bytes = bytes[offset..]
-            .pread_with::<&[u8]>(0, dd.size as usize)
+        let bytes = bytes
```

Comment:
Yea I think this is the correct change to use everywhere, as I’ve mentioned elsewhere. We should probably do a pass looking for patterns like this and convert to full pread as you did here